scanner/src/components/Button/index.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  export let onClick: () => void; 
```

Comment:
Why do we want it to be possible to create buttons without an action?

Reply:
All buttons are required onClick action